internal/service/identitystore/group_test.go

```diff
@@ -0,0 +1,173 @@
+package identitystore_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+
+	sdkacctest "github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-provider-aws/internal/create"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/identitystore"
+	"github.com/aws/aws-sdk-go-v2/service/identitystore/types"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	tfidentitystore "github.com/hashicorp/terraform-provider-aws/internal/service/identitystore"
+	"github.com/hashicorp/terraform-provider-aws/names"
+)
+
+func TestAccIdentityStoreGroup_basic(t *testing.T) {
+	var group identitystore.DescribeGroupOutput
+	resourceName := "aws_identitystore_group.test"
+	displayName := "Acceptance Test"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acctest.PreCheck(t)
+			acctest.PreCheckPartitionHasService(names.IdentityStoreEndpointID, t)
+			testAccPreCheck(t)
+		},
+		ErrorCheck:               acctest.ErrorCheck(t, names.IdentityStoreEndpointID),
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories,
+		CheckDestroy:             testAccCheckGroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccGroupConfig_basic(displayName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGroupExists(resourceName, &group),
+					resource.TestCheckResourceAttr(resourceName, "display_name", displayName),
+					resource.TestCheckResourceAttrSet(resourceName, "identity_store_id"),
+					resource.TestCheckResourceAttrSet(resourceName, "group_id"),
+				),
+			},
+			{
+				ResourceName:      resourceName,
```

Comment:
Note to self: second step, importing seems to fail on a DescribeGroup API call. Need to check this out.

Reply:
Fixed by adjusting the internal resource ID.